src/utils/serve-functions.js

```diff
@@ -121,27 +121,24 @@ function createHandler(dir) {
     }
     const { functionPath } = functions[func]
 
-    const body = request.body.toString()
-    var isBase64Encoded = Buffer.from(body, 'base64').toString('base64') === body
+    const body = request.get('content-length') ? request.body.toString() : undefined
```

Comment:
This makes sure that we're not sending `body` if there is no body. Calling `request.body.toString()` was causing even empty body to produce `[Object]` as string.